.gitignore

```diff
@@ -11,6 +11,5 @@ coverage/
 node_modules/
 examples/test
 .idea/
-package-lock.json
-yarn.lock
-
+*/*/package-lock.json
+*/*/yarn.lock
```

Comment:
Should we need to put yarn.lock in .gitignore? https://github.com/yarnpkg/yarn/issues/1583